third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -1472,15 +1542,29 @@ impl<'a> Resolver<'a> {
                 let field_inst = &module.field_instantiations[idx.0 as usize];
                 let struct_ty = &field_inst.definition_struct_type;
                 let ty_params = &field_inst.instantiation;
-
-                self.loader()
-                    .ty_builder()
-                    .create_struct_instantiation_ty(struct_ty, ty_params, ty_args)
+                self.create_struct_type_instantiation(struct_ty, ty_params, ty_args)
             },
             BinaryType::Script(_) => unreachable!("Scripts cannot have field instructions"),
         }
     }
 
+    pub(crate) fn create_struct_type(&self, struct_ty: &Arc<StructType>) -> Type {
```

Comment:
Let's use `ty` for types? So `create_struct_ty` instead, I think in a couple of other places as well.